nextjs-app/app/components/carousel/CarouselBlock.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { TestimonialCard } from '../testimonial/TestimonialCard'
+import { BlockHeaderSection } from '../BlockHeaderSection'
+import { BackgroundTheme } from '../../types/design-tokens'
+import { Alignment } from '../../types/ui'
+import { CarouselControl } from './CarouselControl'
+import { useHorizontalScroll } from '@/app/hooks/useHorizontalScroll'
+import { CarouselCandidateCard } from './CarouselCandidateCard'
+import { BACKGROUND_COLOR_MAP } from '../../types/design-tokens'
+
+interface CarouselBlockProps {
+  block: {
+    header?: {
+      overline?: string
+      heading: string
+      subhead?: string
+    }
+    variant: 'testimonials' | 'candidates'
```

Comment:
Shouldn't this be an `enum` that can then be reused as a value down below in the control flows referencing it instead of string literals?

<img width="277" alt="image" src="https://github.com/user-attachments/assets/2d57b583-cb92-4ffc-8ea7-5cbe0d1d18cf" />